### DISCUSSION TITLE
How to load groups from Azure SSO

### DISCUSSION THREAD

**Discussion by exander77** [Ylianst/MeshCentral#5836] [Issues]
We have set Azure SSO, but groups are not loading from Azure and I don't see any related options. We would like to set the groups in Azure AD and let MeshCentral just set the membership based on Azure AD groups.

**Comment by si458** [comment]
i dont no how the azure ad works, but you could watch this
https://www.youtube.com/watch?v=_POULGboaCw - 'ldap membership groups' which might help

**Reply by exander77** [reply]
I have seen that, but that's for LDAP, Azure AuthStrategy doesn't have any such options.

**Reply by si458** [reply]
i generally dont no then sadly im afraid, i no LDAP works with windows active directory so i throught it was the same sort of thing

**Reply by exander77** [reply]
I think It may be in the works. I say OICD getting these options added recently. So maybe there just isn't a code that handles this for Azure yet.

**Reply by si458** [reply]
https://github.com/Ylianst/MeshCentral/issues/5179 this maybe?
the was a [PR ](https://github.com/Ylianst/MeshCentral/pull/4530) for OIDC but @Ylianst closed it because it was old and we dont no if it works or not

**Reply by exander77** [reply]
Yes, seems like it. I thought that I could take the idea and adapt it for Azure.

**Reply by si458** [reply]
If you want to have a cracking at adding the code then plz do! Happy to accept PRs
Or
If you can setup a test environment with credentials etc, I can have a play around and see what I can figure out

**Comment by mstrhakr** [comment]
This will be fixed with pull request #5856
